plugins/modules/zos_job_submit.py

```diff
@@ -54,7 +54,7 @@
     type: bool
     description:
       - Wait for the Job to finish and capture the output. Default is false.
-      - User can specify the wait time, see option ``wait_time_s``.
+      - User can specify the wait time, see option ``duration_s``.
```

Comment:
@richp405 - this change reverts a change made 3 months ago,  option `duration_s` does not exist, this concerns me quite a bit because it makes me think that the source/branch/version you are working off of older than 3 months. Please backtrack and try to understand how this happened.

Here was the [merge](https://github.com/ansible-collections/ibm_zos_core/commit/9d9d9297c460ba63c4418e22f161a0892b511f3e) for that